src/Pages/BookAppointment/BookAppointments.jsx

```diff
@@ -0,0 +1,79 @@
+
+import { Spinner } from 'react-bootstrap';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectDoctor } from '../../Redux/Features/doctorSlice';
+import { Link } from 'react-router-dom';
+import './BookAppointment.scss';
+import { getAppointmentDetails } from '../../Redux/Features/appointmentSlice';
+
+
+function BookAppointments() {
```

Comment:
- [ ] I was able to book in the past:

![image](https://github.com/katarighe/book-appointment-front-end/assets/57421823/7193092d-f1bf-4c77-82d5-7f5196d39f5d)

Kindly, add some validation so your users can only book for the future. 👍